support/cas-server-support-syncope-authentication/src/main/java/org/apereo/cas/configuration/model/support/pm/PasswordManagementProperties.java

```diff
@@ -0,0 +1,193 @@
+package org.apereo.cas.configuration.model.support.pm;
+
+import java.io.Serial;
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import org.apereo.cas.configuration.features.CasFeatureModule;
+import org.apereo.cas.configuration.model.core.web.flow.WebflowAutoConfigurationProperties;
+import org.apereo.cas.configuration.model.support.captcha.GoogleRecaptchaProperties;
+import org.apereo.cas.configuration.support.RequiresModule;
+import org.springframework.boot.context.properties.NestedConfigurationProperty;
+
+/**
+ * This is {@link PasswordManagementProperties}.
+ *
+ * @author Misagh Moayyed
+ * @since 5.0.0
+ */
+@RequiresModule(name = "cas-server-support-pm-webflow")
+public class PasswordManagementProperties implements CasFeatureModule, Serializable {
```

Comment:
I don't think you need this; already exists. You just need to add the syncope set of settings.